proto/config/edge_node_cluster.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright(c) 2024 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+package org.lfedge.eve.config;
+option go_package = "github.com/lf-edge/eve-api/go/config";
+option java_package = "org.lfedge.eve.config";
+
+message EdgeNodeCluster {
+  // cluster name, in case it has multiple cluster on the same site
+  string ClusterName = 1;
+
+  // cluster UUID
+  string ClusterID = 2;
+
+  // Cluster-Interface
```

Comment:
Please provide an example in the comment e.g., "eth1".

Reply:
Done,  changed the comment to
```go
// Cluster-Interface, for example "eht1"
```